kwc-challenge-card.html

```diff
@@ -9,26 +13,134 @@
 
 <dom-module id="kwc-challenge-card">
   <template>
+
     <style>
-      :host {
-        display: block;
-      }
+        :host {
+            @apply --layout-vertical;
+            border-radius: 3px;
+            background-color: white;
+            overflow: hidden;
+            transform: translateZ(0);
+        }
+        :host .kwc-cc {
+          width: var(--kwc-cc-width, 184px);
+          margin: auto;
+          cursor: pointer;
+        }
+        :host .title,
+        .card-content {
+            color: var(--color-black);
+            margin: 0;
+            font-size: 12px;
+            text-decoration: none;
+        }
+        :host .title {
+            font-size: 16px;
+            font-weight: bold;
+        }
+        :host .cover,
+        :host .cover > iron-image {
+            background-color: inherit;
+            width: 100%;
+            z-index: 0;
+            height: var(--kwc-cc-cover-height, 176px);
+        }
+        :host .cover {
+            position: relative;
+        }
+        :host .info {
+            @apply --layout-flex;
+            border-width: 0 1px 1px 1px;
+            border-style: solid;
+            border-color: var(--color-grey-lighter);
+            position: relative;
+        }
+        :host .header {
+            @apply --layout-horizontal;
+            @apply --layout-justified;
+            @apply --layout-center;
+            padding: 12px 8px 8px 8px;
+        }
+        :host .tick {
+            color: white;
+            background-color: var(--color-sushi);
+            border-radius: 100%;
+            height: 12px;
+            padding: 4px;
+            width: 12px;
+        }
+        :host([completed]) .kwc-cc {
+            background-color: var(--color-grassland);
+        }
+        :host([completed]) .title,
+        :host([completed]) .card-content {
+            color: white;
+        }
+        :host([completed]) .info {
+            border-color: var(--color-grassland);
+        }
+        :host[hidden] {
+            display: none !important;
+        }
+        @media screen and (max-width: 680px) {
+            :host {
+                width: 285px;
+            }
+        }
     </style>
-    <h2>Hello [[prop1]]!</h2>
+    <div class="kwc-cc-container">
+      <div class="kwc-cc">
+        <div class="cover">
+            <iron-image style="background-color: lightgray;"
+              src="[[imageurl]]" sizing="cover" preload fade></iron-image>
+        </div>
+        <div class="info">
+          <div class="header">
+            <h4 class="title" id="title">{{title}}</h4>
+              <template is="dom-if" if="[[completed]]">
+                <iron-icon class="tick" icon="kano-icons:tick"></iron-icon>
+              </template>
```

Comment:
It looks like we need to wrap this in  `<dom-if></dom-if>` to make this a hybrid component